src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.FSharp.props

```diff
@@ -0,0 +1,35 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.FSharp.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- If any of these variables are set then we are building under the fsharp.sdk so use their logic -->
+  <PropertyGroup Condition = "'$(_FscTaskAssemblyPath_net)' == '' and '$(_FscTaskAssemblyPath_netcoreapp1_0)' == '' and '$(_FscToolFullPath_net)' == '' and '$(_FscToolFullPath_netcoreapp1_0)' == ''">
+    <NotBuildingUnderFSharpSdk>true</NotBuildingUnderFSharpSdk>
+  </PropertyGroup>
+
+  <Choose>
+    <When Condition=" '$(MSBuildRuntimeType)' == 'core' ">
+      <PropertyGroup>
+         <FSharpPropsShim>$(MSBuildToolsPath)\FSharp\Microsoft.NET.Sdk.FSharp.props</FSharpPropsShim>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <FSharpPropsShim>$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.NET.Sdk.FSharp.props</FSharpPropsShim>
```

Comment:
This file is the existing VS shim to the f# compiler.   The Microsoft.NET.Sdk.FSharp.props that ships with the SDK is the SDK shim that switches between dotnet cli deployed compiler and the desktop compiler dependent on which msbuild is running.